common/reviews/api/rush-lib.api.md

```diff
@@ -1398,6 +1398,12 @@ export class RushLifecycleHooks {
 export class RushProjectConfiguration {
     readonly disableBuildCacheForProject: boolean;
     getCacheDisabledReason(trackedFileNames: Iterable<string>, phaseName: string, isNoOp: boolean): string | undefined;
+    static getCacheDisabledReasonForProjectAsync(project: RushConfigurationProject, options: {
+        terminal: ITerminal;
```

Comment:
Any suggestions for a better API here welcome, I don't love mixing between named options and positional parameters, but I wanted to make it clear this is for a specific project.

Reply:
I really don't like passing an `ITerminal` to this function, or having it be async; I refactored it into its own function _specifically_ to have it return a synchronous result that could be separately logged at the whims of the caller.

Reply:
Looks like you need to rebuild to regenerate the api.md

Reply:
Should be updated in the latest commit 😄